sql/src/main/java/org/elasticsearch/action/bulk/BulkShardProcessor.java

```diff
@@ -225,6 +225,7 @@ private ShardId shardId(String indexName, String id, @Nullable String routing) {
                 routing
             ).shardId();
         } catch (IndexNotFoundException e) {
+            LOGGER.warn(String.format(Locale.ENGLISH, "Trying to insert value into unavailable partition: %s", indexName));
```

Comment:
Shouldn't we only log if autoCreateIndices is false? Otherwise we log even though we're going to create the index?